src/components/PageHeader/index.js

```diff
@@ -18,7 +15,7 @@ const PageHeader = ({ description, children, contentType }) => (
         <p className="coa-PageHeader__description">{description}</p>
       )}
 
-      {!contentType && !contentType === 'service' ? <SectionHeader /> : null}
+      {contentType && contentType === 'information' ? <SectionHeader /> : null}
```

Comment:
I think it'd be good to do a bit of digging into how we want the logic behind showing/hiding SectionHeader to work. Overkill to implement now but I made an issue for the backlog (https://github.com/cityofaustin/techstack/issues/1552) so we can address it in the future.